packages/web/src/styles/Button.js

```diff
@@ -3,7 +3,7 @@ import { css } from '@emotion/core';
 import styled from '@emotion/styled';
 import { darken, lighten, rgba } from 'polished';
 
-const filters = ({ colors: { borderColor } }) => css`
+const Filter = () => css`
```

Comment:
Just confirming if Removing the param here would nullify the effect of 

https://github.com/appbaseio/reactivesearch/pull/2248/files#diff-1cd4e1f165357548d76d83cd7b7d61ccbb3050095c70e53832a0f0d11ef0b5abR201

Reply:
No.

Reply:
@SavvyShah  Can you share a demo of this component being utilized with different themes - light/ dark?

Reply:
Check comments above